stable/opsportal/values.yaml

```diff
@@ -68,23 +68,18 @@ opsportal:
 ###############################################################################
 #  Konvoy UI
 ###############################################################################
-kommander:
```

Comment:
I have to say, this change might cause some issues to running clusters :/. 

Scenario: Imagine we have a cluster where the customer changed the values property for the opsportal addon.
```
 kommander:
   yada: 
   yada
```

With this rename, the new changes might not be applied to konvoy-ui. Can we double check that ?

Reply:
Otherwise, we should inform the customers that they need to update the `values: ` for opsportal from kommander to konvoy-ui.